packages/code-du-travail-frontend/src/outils/common/Feedback/QuestionnaireAdvanced.tsx

```diff
@@ -0,0 +1,115 @@
+import { Button, Heading } from "@socialgouv/cdtn-ui";
+import { QuestionnaireItem } from "./QuestionnaireItem";
+import { QuestionnaireText } from "./QuestionnaireText";
+import styled from "styled-components";
+import { useState } from "react";
+import {
+  trackFeedback,
+  trackFeedbackText,
+  FEEDBACK_RESULT,
+  EVENT_ACTION,
+} from "./tracking";
+import { useRouter } from "next/router";
+
+type QuestionnaireAdvancedProps = {
+  onClick: () => void;
+};
+
+export const QuestionnaireAdvanced = ({
+  onClick,
+}: QuestionnaireAdvancedProps): JSX.Element => {
```

Comment:
JSX.Element est deprecated (https://stackoverflow.com/questions/76224848/what-is-the-correct-return-type-replacement-for-jsx-element-after-the-global-jsx)

```suggestion
}: QuestionnaireAdvancedProps): React.ReactElement => {
```